plugin/src/main/java/io/snyk/plugins/nexus/util/Formatter.java

```diff
@@ -66,15 +66,57 @@ public static void enrichScanResultWithLicenseIssues(@Nonnull ScanResult scanRes
       return;
     }
 
-    String[] parts = formattedIssues.split(", ");
+    Optional<IssueCounts> issueCounts = getIssueCountsFromFormattedString(formattedIssues);
+    if (issueCounts.isPresent()) {
+      // regardless of if critical is present or not, we ignore it
+      scanResult.highLicenseIssueCount = issueCounts.get().high;
+      scanResult.mediumLicenseIssueCount = issueCounts.get().medium;
+      scanResult.lowLicenseIssueCount = issueCounts.get().low;
+    } else {
+      throw new RuntimeException(String.format("Invalid format for license issues: %s", formattedIssues));
+    }
+  }
 
-    String high = parts[0].replace(" high", "");
-    scanResult.highLicenseIssueCount = Long.parseLong(high);
+  public static Optional<IssueCounts> getIssueCountsFromFormattedString(String formattedIssues) {
+    Matcher issuesWithCriticalMatch = ISSUES_WITH_CRITICAL_REGEX.matcher(formattedIssues);
+    if (issuesWithCriticalMatch.find()) {
+      return Optional.of(new IssueCounts(
+        issuesWithCriticalMatch.group(1),
+        issuesWithCriticalMatch.group(2),
+        issuesWithCriticalMatch.group(3),
+        issuesWithCriticalMatch.group(4)
+      ));
+    }
+
+    Matcher issuesWithoutCriticalMatch = ISSUES_WITHOUT_CRITICAL_REGEX.matcher(formattedIssues);
```

Comment:
Why do you use `matches` here, and `find` for the Critical Matcher?

Reply:
Mistake (thought they both work) - I switched them to both use `matches()`.
Thanks